api/urls.py

```diff
@@ -75,13 +75,21 @@ def to_url(self, value):
     api_router.register(r"relaynumber", RelayNumberViewSet, "relay_number")
     api_router.register(r"inboundcontact", InboundContactViewSet, "inbound_contact")
     urlpatterns += [
+        # TODO: Update Twilio webhooks to versions with trailing slashes,
+        #       then remove versions without trailing slashes (Django's
+        #       APPEND_SLASH option will then make those redirect).
         path("v1/inbound_sms", inbound_sms, name="inbound_sms"),
+        path("v1/inbound_sms/", inbound_sms, name="inbound_sms"),
         path("v1/inbound_call", inbound_call, name="inbound_call"),
+        path("v1/inbound_call/", inbound_call, name="inbound_call"),
         path("v1/voice_status", voice_status, name="voice_status"),
+        path("v1/voice_status/", voice_status, name="voice_status"),
         path("v1/sms_status", sms_status, name="sms_status"),
+        path("v1/sms_status/", sms_status, name="sms_status"),
         path("v1/vCard/<lookup_key>", vCard, name="vCard"),
+        path("v1/vCard/<lookup_key>/", vCard, name="vCard"),
```

Comment:
This one is a `GET`, so we might be able to remove the version without a trailing slash. Or, we could wait until after the next deploy.